File: Algoritmos Gulosos/Item.java

public class Item {
	String descricao;
	double peso;
	double valor;
	double media;
	
	public Item(String descricao, double peso, double valor) {
		this.descricao = descricao;
		this.peso = peso;
		this.valor = valor;
		this.media = valor/peso;
	}

	public String getDescricao() {
		return descricao;
	}

	public void setDescricao(String descricao) {
		this.descricao = descricao;
	}

	public double getPeso() {
		return peso;
	}

	public void setPeso(double peso) {
		this.peso = peso;
	}

	public double getValor() {
		return valor;
	}

	public void setValor(double valor) {
		this.valor = valor;
	}

	public double getMedia() {
		return media;
	}

	public void setMedia(double media) {
		this.media = media;
	}
	
	

}

File: ALGORTIMO_MOCHILA/Item.py
#ALGORITMO ITEM
class Item:
    def __init__(self, nome, peso, valor):
        self.__nome = nome
        self.__peso = peso
        self.__valor = valor
    
    def getNome(self):
        return self.__nome
    def getPeso(self):
        return self.__peso
    def getValor(self):
        return self.__valor
    def getMedia(self):
        return self.__valor/self.__peso